pkg/database/orm/orm.go

```diff
@@ -73,19 +73,39 @@ func (o *ORM) Initialize() error {
 			o.config.DbUser,
 			o.config.DbPassword,
 			o.config.ConnectionParams)
+		dbDialector = postgres.Open(o.connectionStr)
 	}
 
-	opened, err := gorm.Open(databaseType, o.connectionStr)
+	opened, err := gorm.Open(dbDialector, &gorm.Config{
+		SkipDefaultTransaction:                   false,
+		NamingStrategy:                           nil,
+		FullSaveAssociations:                     false,
+		Logger:                                   nil,
+		NowFunc:                                  nil,
+		DryRun:                                   false,
+		PrepareStmt:                              false,
+		DisableAutomaticPing:                     false,
+		DisableForeignKeyConstraintWhenMigrating: false,
+		AllowGlobalUpdate:                        false,
+		ClauseBuilders:                           nil,
+		ConnPool:                                 nil,
+		Dialector:                                nil,
+		Plugins:                                  nil,
+	})
 	if err != nil {
 		o.log.Logger.Error(err, "Could not open database.")
 		return errors.Wrap(err, "failed to open database")
 	}
 
-	opened.SetLogger(o.logger)
-	opened.LogMode(o.enableLog)
-
 	o.db = opened
-	o.db.DB().SetConnMaxLifetime(time.Hour)
+	sqldb, err := o.db.DB()
+	if err != nil {
+		err = errors.WithMessage(err, "Could not access sub sql database.")
+		o.log.Logger.Error(err)
+		return err
+	}
+	o.sqldb = sqldb
+	o.sqldb.SetConnMaxLifetime(time.Hour)
```

Comment:
not a big deal but one hour feels a bit short

Reply:
kept from previous settings but should I change to something else ?
or add it into the config file ?

Reply:
Having it in the config file seems like a good idea IMHO.

Reply:
Done